src/mesido/workflows/multicommodity_simulator_workflow.py

```diff
@@ -0,0 +1,624 @@
+import locale
+import logging
+import time
+from pathlib import Path
+
+import esdl
+
+from mesido.esdl.esdl_mixin import ESDLMixin
+from mesido.esdl.esdl_parser import ESDLFileParser
+from mesido.esdl.profile_parser import ProfileReaderFromFile
+from mesido.head_loss_class import HeadLossOption
+from mesido.physics_mixin import PhysicsMixin
+from mesido.workflows.io.write_output import ScenarioOutput
+from mesido.workflows.utils.helpers import main_decorator
+
+import numpy as np
+
+from rtctools.optimization.collocated_integrated_optimization_problem import (
+    CollocatedIntegratedOptimizationProblem,
+)
+from rtctools.optimization.goal_programming_mixin import Goal
+from rtctools.optimization.linearized_order_goal_programming_mixin import (
+    LinearizedOrderGoalProgrammingMixin,
+)
+from rtctools.optimization.single_pass_goal_programming_mixin import (
+    CachingQPSol,
+    SinglePassGoalProgrammingMixin,
+)
+from rtctools.util import run_optimization_problem
+
+DB_HOST = "172.17.0.2"
+DB_PORT = 8086
+DB_NAME = "Warmtenetten"
+DB_USER = "admin"
+DB_PASSWORD = "admin"
+
+logger = logging.getLogger("WarmingUP-MPC")
+logger.setLevel(logging.INFO)
+
+locale.setlocale(locale.LC_ALL, "")
+
+WATT_TO_MEGA_WATT = 1.0e6
```

Comment:
I like your thought, particularly since it is also used in other workflows, I'll create an issue for it. (I would then like to adjust them all together.)